packages/plugin-core/src/test/testUtilsV3.ts

```diff
@@ -654,7 +655,10 @@ export function cleanupWorkspaceStubs(ctx: ExtensionContext): void {
 export function subscribeToEngineStateChange(
   callback: (noteChangeEntries: NoteChangeEntry[]) => void
 ): Disposable {
-  const engine = ExtensionProvider.getEngine();
-  const engineClient = engine as unknown as DendronEngineClient;
+  const engineClient = toDendronEngineClient(ExtensionProvider.getEngine());
   return engineClient.onEngineNoteStateChanged(callback);
 }
+
+export function toDendronEngineClient(engine: IEngineAPIService) {
```

Comment:
Hrm - outside of the scope of this change, but this casting shouldn't be necessary if we fix our interfaces.  `IEngineAPIService` should extend `DendronEngineClient`

Reply:
Makes sense to me, although we'll need to talk about the howtos, especially since the constructors are pretty different